packages/headless/src/features/search-parameters/search-parameter-actions.ts

```diff
@@ -16,7 +20,8 @@ export type SearchParameters = Omit<
   | 'searchHub'
   | 'pipeline'
   | 'context'
->;
+> &
+  Partial<FacetParameters>;
```

Comment:
Why partial here ?

Reply:
The `f` key may not be present if there are no facets, or no selected values. Another option could be to add the `?` next to the `f` key. I am fine with either approach,